common/types/platform-bufferutils.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module 'platform-bufferutils' {
```

Comment:
I'm a bit cautious about using `data` because `buffer` is more precise, but I understand the preference to avoid abbreviations so I've changed all buffer operations to use `buffer` in 3891c09.